mysql-test/include/restart_mysqld_with_invalid_option.inc

```diff
@@ -0,0 +1,35 @@
+
+if ($rpl_inited)
+{
+  if (!$allow_rpl_inited)
+  {
+    --die ERROR IN TEST: This script does not support replication
+  }
+}
+
+# Write file to make mysql-test-run.pl expect the "crash", but don't start
```

Comment:
If you keep this include file, then this snippet should be replaced by source shutdown_mysqld.inc (unless this happens to break caller's replace_result - in which case at least the explicit "10" should be removed from shutdown_server so that the default mysqltest timeout becomes effective)